plugins/circuitbreaker/README.md

```
@@ -0,0 +1,201 @@
+                    },
+                },
+            },
+        },
+    })
+}
+```
+
+### State Diagram of Circuit Breaker
+![Circuit Breaker States](../../docs/media/plugins/circuit-breaker-states.png)
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)